backend/app/tests/api/routes/collections/test_create_collections.py

```
@@ -60,8 +64,14 @@
             "instructions": "Test collection assistant.",
             "temperature": 0.1,
         }
+        
         headers = normal_user_api_key_headers
 
+        mock_get_credential.return_value = {"api_key": "test_api_key"}
```

Ruff (0.12.2)

Undefined name `mock_get_credential`

(F821)